### PR TITLE
Small changes in OPM Met report

### PR DESCRIPTION
**Pull request by lwyszomi**
@esoergel @sravfeyn

### CODE REVIEW

File: custom/opm/reports.py [hunk 1]
```diff
@@ -893,31 +894,31 @@ def row_objects(self):
...
-                    })
+                    self.add_dubug_data(row._id, e)
+
         self.total_row = ["" for __ in self.model.method_map]
         self.total_row[0] = _("Total Payment")
-        self.total_row[self.column_index('cash_pay')] = "Rs. {}".format(total_payment)
+        self.total_row[self.column_index('cash')] = "Rs. {}".format(total_payment)
         return rows
 
+    def add_dubug_data(self, row_id, e ):
```
Comment (dimagimon) [inline custom/opm/reports.py]: E202 whitespace before ')'

**Comment by lwyszomi** [comment]
@sravfeyn Updated

File: custom/opm/beneficiary.py [hunk 2]
```diff
@@ -858,6 +857,49 @@ def __init__(self, case, report, child_index=1, awc_codes={}, **kwargs):
...
+        self.five = ''
+        self.payment_last_month = ''
+        self.issue = _('Reporting period incomplete')
+        if self.status == 'mother':
+            self.child_name = self.case_property(self.child_xpath("child{num}_name"), EMPTY_FIELD)
+        elif self.status == 'pregnant':
+            self.child_name = EMPTY_FIELD
+
+    @property
+    def readable_status(self):
```
Comment (sravfeyn) [inline custom/opm/beneficiary.py]: Can this be removed, it looks identical to the parent-class.

**Comment by lwyszomi** [comment]
@sravfeyn Updated

**Comment by sravfeyn** [comment]
Thanks. (waiting for tests)

**Comment by esoergel** [comment]
This is causing http://manage.dimagi.com/default.asp?175215
If using exceptions to catch these cases is too messy, we could switch to using a flag on the model - initialize it with `case_is_out_of_range` set to `False`, then rather than raising `CaseOutOfRange`, set `case_is_out_of_range` to `True`, then raise `CaseOutOfRange` later if that flag is set.  This will allow the other properties to be calculated.  No strong feelings about how to implement this, though.

**Pull request merged**